packages/fast-form-generator-react/README.md

```diff
@@ -0,0 +1,181 @@
+
+# ReactJS FAST form generator
+A self generating UI based on JSON Schemas.
+
+It can be utilized to dynamically generate a form for changing data values on a React component.
+
+## Installing
+- Run `npm i`
+
+## Getting started
+- Run `npm start`
```

Comment:
not `npm run start`? It is like that everywhere else

Reply:
actually dont need to say run, its like some of the other scripts that npm can recognise